### PR TITLE
docs: update `port` example

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [X] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

NA

### Motivation / Use-Case

Update docs.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: examples/cli/host-and-port/README.md [hunk 1]
```diff
@@ -3,46 +3,79 @@
...
 2. You should see the text on the page itself change to read `Success!`.
+
+## port
+
+### specific port
+
+Tell the server to connect to a specific port with the following:
+
+```console
+npx webpack serve --open-target --port 9000
```
Comment (anshumanv) [inline examples/cli/host-and-port/README.md:59]: just open should do

Comment (snitin315) [reply examples/cli/host-and-port/README.md:59]: `--open` is deprecated and will be removed in v5 in favor of `--open-target`.

Comment (anshumanv) [reply examples/cli/host-and-port/README.md:59]: oh yep

**Review by anshumanv — APPROVED** [review]
Let @alexander-akait also take a look

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**